flocx_market/matcher.py

```diff
@@ -89,12 +92,25 @@ def apply_operator(val1, val2, op):
 
 def match_specs(match_expression, data):
     for exp in match_expression:
-        if None in exp:
-            return False
 
         j_val = jmespath.search(exp[0], data)
         op = exp[1]
         val = exp[2]
+
         if not apply_operator(j_val, val, op):
             return False
+
     return True
+
+
+def get_all_offers_matching_specs(specs):
+
+    all_offers = OfferList.get()['offers']
```

Comment:
This is probably not the right way to get the list of offers (because the `OfferList` class is explicitly for implementing the REST API).  With the current code, you probably want to use the `OfferApi` class, as in `OfferApi.query.all()`.